apps/website/src/examples/index.tsx

```diff
@@ -146,7 +146,14 @@ export { default as ListSubcomponentsExample } from './List.subcomponents';
 export { default as ListLinksExample } from './List.links';
 export { default as ListComboboxExample } from './List.combobox';
 
-export { default as NonIdealStateMainExample } from './NonIdealState.main';
+export { default as NonIdealStateBadgateawayExample } from './NonIdealState.badgateaway';
+export { default as NonIdealStateForbiddenExample } from './NonIdealState.forbidden';
+export { default as NonIdealStateInternalErrorExample } from './NonIdealState.internalerror';
+export { default as NonIdealStatePagenotfoundExample } from './NonIdealState.Pagenotfound';
+export { default as NonIdealStateRedirectExample } from './NonIdealState.redirect';
+export { default as NonIdealStateServiceunavailableExample } from './NonIdealState.serviceunavailable';
+export { default as NonIdealStateTimeoutExample } from './NonIdealState.timeout';
+export { default as NonIdealStateUnauthorizedExample } from './NonIdealState.unauthorized';
```

Comment:
You forgot to include this one.
```suggestion
export { default as NonIdealStateUnauthorizedExample } from './NonIdealState.unauthorized';
export { default as NonIdealStateErrorExample } from './NonIdealState.error';
```